sfa_dash/static/js/report-form-utilities.js

```diff
@@ -202,23 +202,35 @@ report_utils.searchObjects = function(object_type, object_id){
     return metadata;
 }
 
-report_utils.setVariables = function(){
-	/* Displays or hides options in the variable <select> element based on the
-     * current units.
+report_utils.setVariables = function(variable=null){
+	/*  Displays or hides options in the variable <select> element based on the
+     *  current units.
+     *
+     *  @param {string} variable
+     *      The variable to remain selected after filtering. Ignored when
```

Comment:
The variable to remain? Multiple variables remain if they share units.

Reply:
Yes, but for example after adding a pair with variable 'ghi', this ensures that 'ghi' is what remains in the variables filter after filtering takes place. This does change the select from "All Variables" to that value, which I can see is somewhat confusing.

We could allow "All Variables" to remain an option while filtering the forecasts by the current units but that would take some more work. The way filtering works now, forecasts are filtered by the currently selected variable, and the options for the variable filter are determined by units. Hopefully that makes sense.